rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md

```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
 
 Using your mailbox as a task list also saves you from having to use inefficient paper-based cards to track your tasks.
 
-::: email-template  
+::: email-template\
```

Comment:
Please remove the trailing slash's in this file  i.e.`\`  (This is a bug with Netlify 😟)